std/experimental/ndslice/internal.d

```diff
@@ -1,3 +1,5 @@
+/// @@@DEPRECATED_2017-04@@@
+deprecated("Please use mir-algorithm DUB package: http://github.com/libmir/mir-algorithm")
```

Comment:
Okay - would you be so kind to create it, s.t. deprecation warnings don't go to a nirvana?